### PR TITLE
Update troubleshooting doc for video previews

### PR DESCRIPTION
**Pull request by jasonblais**
Address feedback from https://github.com/mattermost/docs/pull/981/files/5e67ab50bac5a3ddc5febe76facfbf97a9c498d2#diff-17710e4b6b10a0dde1fdc4f58c58ce34

### CODE REVIEW

File: source/install/troubleshooting.rst [hunk 1]
```diff
@@ -63,7 +63,7 @@ If email sign-in was turned off before the System Administrator switched sign-in
 Unable to play audio or video files on browser
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Your server may be misconfigured where the MIME type isn't set correctly. Try installing a media player or read the `related documentation <https://golang.org/pkg/mime/#TypeByExtension>`_ to add the missing MIME types.
+The server might not be using the correct MIME type for the media files. To configure MIME types on your server, see `related documentation <https://golang.org/pkg/mime/#TypeByExtension>`_.
```
Comment (JeffSchering) [inline source/install/troubleshooting.rst]: The page that you link to says nothing about how to configure MIME types on a web server.

Comment (JeffSchering) [inline source/install/troubleshooting.rst]: What is the correct MIME type for the media files?

Comment (jasonblais) [inline source/install/troubleshooting.rst]: @JeffSchering I google searched and found some relevant docs
 - Ubuntu: https://help.ubuntu.com/community/AddingMimeTypes
 - Windows IIS 7: https://technet.microsoft.com/en-us/library/cc725608(v=ws.10).aspx

Comment (jasonblais) [inline source/install/troubleshooting.rst]: Should we give these two as examples, given there's probably a separate page for different server configurations.

I also suspect the correct MIME types will depend on environment

Comment (jasonblais) [inline source/install/troubleshooting.rst]: @JeffSchering  Following our discussion yesterday, propose removing this section from install guides, given

 - it should only affects Windows servers, which we don't officially support
 - no one has actually reported the issue yet
 - we're not certain what exact steps are needed to troubleshoot and resolve the issue

**Pull request merged**